client/components/frontpage/ChinguSection.js

```diff
@@ -107,7 +106,6 @@ const ChinguSection = () => {
       <ContentWrapper>
         <ImageSection>
           <Logo src="/home-chingu-image.png" />
```

Comment:
While you're on it, could you please rename this component? The picture we use is not a Logo, so it should ideally have a different naming.

Reply:
Hey, I just renamed the component and added some padding to the image, please take a look when you have time. And I'll keep in mind to comment under issues before I do them in the future, sorry about that.

Reply:
No problem, thank you!